content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Multi-cluster visibility with federation"
+linkTitle: "Reaching Multi-cluster Visibility"
+description: "In this guide, you'll learn how to register external clusters using the federation API and access resources across multiple clusters in the web UI."
+weight: 400
+version: "5.15"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.15:
+    parent: guides
+---
+
+- [What you can do with federation](#what-you-can-do-with-federation)
+- [Step 1 Register clusters](#step-1-register-clusters)
+  - [Register a single cluster](#register-a-single-cluster)
+  - [Register additional clusters](#register-additional-clusters)
+- [Step 2 Set up communication between clusters](#step-2-set-up-communication-between-clusters)
+- [Step 3 Create etcd replicators](#step-3-create-etcd-replicators)
```

Comment:
These links are broken as well. I suppose there is some reordering that needs to be done in this overview.